eva/udfs/ndarray/annotate.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import cv2
+import numpy as np
+import pandas as pd
+
+from eva.udfs.abstract.abstract_udf import AbstractUDF
+
+color = (207, 248, 64)
+thickness = 4
+
+
+class Annotate(AbstractUDF):
```

Comment:
Please use `decorators` to write this UDF.  Also, it is unclear what are the inputs and outputs of the UDF.